### ISSUE TITLE
Disable RVM status area

### ISSUE THREAD

**Issue by iainbeeston** [davidjrice/prezto_powerline#5]
At the moment, prezto power line is a bit broken for anyone not using rvm. Anyone not using ruby is left with a useless white block at the right of the prompt. Anyone using ruby but not rvm (for example chruby or rbenv) would still find this block useless.

Would it be possible to add a config variable to disable that area? Or simply not show it if RVM is not installed?

**Comment by davidjrice** [comment]
Yeah, I’m all in favour of this, it’s crossed my mind a few times but I haven’t had the time. It would also be good to support npm etc. Just need to find a good way to achieve it without compromising performance.

**Comment by blcarson** [comment]
@iainbeeston removing lines 177 to 179 did the trick for me
